packages/server/tests/helpers/utils.ts

```diff
@@ -308,4 +309,61 @@ export class Utils {
     }
     return accounts;
   }
+
+  static sendJsonRpcRequestWithDelay(
+    host: string,
+    port: number,
+    method: string,
+    params: any[],
+    delayMs: number,
+  ): Promise<any> {
+    const requestData = JSON.stringify({
+      jsonrpc: '2.0',
+      method: method,
+      params: params,
+      id: 1,
+    });
+
+    const options = {
+      hostname: host,
+      port: port,
+      path: '/',
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        'Content-Length': Buffer.byteLength(requestData),
+      },
+      timeout: delayMs,
+    };
+
+    return new Promise((resolve, reject) => {
+      const req = http.request(options, (res) => {
```

Comment:
Maybe some short comments along this process to explain each step?

Reply:
Done

Reply:
Done